Week2/README.md

```diff
@@ -216,44 +229,44 @@ Check out the following to get a more visual idea of what `joins` are:
 
 In database management an **Aggregate Function** is a function where the values of multiple rows are grouped together as input on certain criteria to form a single value of more significant meaning.
 
-- `Count()`
-- `Sum()`
-- `Avg()`
-- `Min()`
-- `Max()`
+- `COUNT()`
+- `SUM()`
+- `AVG()`
+- `MIN()`
+- `MAX()`
 
 ### Distinct Keyword
 
 DISTINCT statement is used to return only distinct (different) values. It can be used with aggregation functions. In below example, we retrieve the numbers of teachers from students table.
 
 ```sql
-SELECT Count(DISTINCT s.teacher_number) AS no_teachers
-FROM students s
+SELECT COUNT(DISTINCT teacher_number) AS no_teachers
+FROM students
 ```
 
 ### Group By
 
-The **GROUP BY** statement groups rows that have the same values into summary rows, like "find the number of students for each teacher".
+The `GROUP BY` statement groups rows that have the same values into summary rows, like "find the number of students for each teacher".
 
-The **GROUP BY** statement is often used with aggregate functions to group the result-set by one or more columns.
+The `GROUP BY` statement is often used with aggregate functions to group the result-set by one or more columns.
 
 ```sql
-SELECT Count(s.techer_number) AS no_teachers, s.techer_number AS teacher_number
-FROM students s
-GROUP BY s.teacher_number
+SELECT COUNT(techer_number) AS no_teachers, techer_number AS teacher_number
```

Comment:
Nitpick: spelling mistake from earlier query is repeated. techer=> teacher

Reply:
Good catch. I fixed the typo.